src/Std/Tactic/BVDecide/Normalize/BitVec.lean

```diff
@@ -362,6 +362,7 @@ attribute [bv_normalize] BitVec.umulOverflow_eq
 attribute [bv_normalize] BitVec.smulOverflow_eq
 attribute [bv_normalize] BitVec.usubOverflow_eq
 attribute [bv_normalize] BitVec.ssubOverflow_eq
+attribute [bv_normalize] BitVec.sdivOverflow_eq
```

Comment:
Do we lack some test cases for bv_normalize?

Reply:
added one!